opal/mca/threads/pthreads/Makefile.am

```diff
@@ -30,3 +30,5 @@ libmca_threads_pthreads_la_SOURCES = \
     threads_pthreads_tsd.h \
     threads_pthreads_wait_sync.c \
     threads_pthreads_wait_sync.h
+
+AM_LDFLAGS = -labt
```

Comment:
I don't think this is right.  Why add -labt for pthreads specific Makefile.am?

Reply:
You are right. It is wrong. I will fix it soon.

Reply:
I wrongly put it in `pthreads/Makefile.am`. This `AM_LDFLAGS` was moved to `argobots/Makefile.am`.